serializer.js

```diff
@@ -69,36 +69,36 @@ module.exports = class Serializer {
     return bool === null ? 'null' : this.asBoolean(bool)
   }
 
-  asDatetime (date) {
-    const quotes = '"'
+  asDateTime (date) {
+    if (date === null) return '""'
     if (date instanceof Date) {
-      return quotes + date.toISOString() + quotes
+      return '"' + date.toISOString() + '"'
     }
-    return this.asString(date)
+    throw new Error(`The value "${date}" cannot be converted to a date-time.`)
```

Comment:
What if I passed a string that fulfill `date-time`?
It seems like it only accept `Date` object after this change.

Reply:
If you pass the string it will be stringified in the `asString` method.